apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -910,6 +910,20 @@ describe('LIVECHAT - rooms', function () {
 				.expect(400);
 		});
 
+		it('should throw and error if file uploads are disabled', async () => {
```

Comment:
Can we also test cases for:

- When `FileUpload_Enabled` is enabled and `Livechat_fileupload_enabled` is enabled too
- When `FileUpload_Enabled` is disabled and `Livechat_fileupload_enabled` is disabled too
- When `FileUpload_Enabled` is enabled and `Livechat_fileupload_enabled` is disabled
- When `FileUpload_Enabled` is disabled and `Livechat_fileupload_enabled` is enabled